lighthouse-core/test/audits/is-on-https-test.js

```diff
@@ -30,7 +30,6 @@ describe('Security: HTTPS audit', () => {
     ]), {computedCache: new Map()}).then(result => {
       assert.strictEqual(result.score, 0);
       expect(result.displayValue).toBeDisplayString('2 insecure requests found');
-      assert.strictEqual(result.details.items.length, 2);
```

Comment:
`result.details.items.length` is still 2.  Are you just removing this because the line above captures the same information?

Reply:
oops, I meant to remove `expect(result.details.headings).toHaveLength(2);`.